cronjob/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: Helm chart for deploying an instance of a cronjob on a K8s cluster.
+name: cronjob
+version: 0.0.1
```

Comment:
i think a version like `1.0.0` should be used as starting point

Reply:
https://github.com/commercetools/k8s-charts/pull/1/commits/a69e63c61251e1123fa4cb6ac206bc7fa1aeb222